src/services/storage/index.js

```diff
@@ -0,0 +1,70 @@
+import fs from 'fs';
```

Comment:
Too complex for my taste. Don't know why you need two classes here.

Reply:
Fixed in task 5